src/index.html

```diff
@@ -1,13 +1,59 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="root">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin=" "
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="card" data-qa="card">
+      <div class="card__image"></div>
+
+      <div class="card__text">
+        <h4 class="card__text card__text--name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h4>
+
+        <span class="card__text card__text--product-code">
+          Product code: 195434
+        </span>
```

Comment:
Consider will be better you use a tag p here instead of span

Reply:
`&--4 > :nth-child(-n + 4) {
     background-image: url("../images/star-active.svg");
   }` I corrected it